src/components/cylc/workflow/Lumino.vue

```diff
@@ -151,6 +151,7 @@ export default {
      * }}
      */
     onWidgetActivated (customEvent) {
+      // eslint-disable-next-line vue/custom-event-name-casing
       this.$emit('lumino:activated', customEvent.detail)
```

Comment:
Funny, the docs seem to suggest that this pattern is correct?

https://eslint.vuejs.org/rules/custom-event-name-casing.html#rule-details

Reply:
Yup! I guess that's a bug in their linter rule. Another rule failed too, where we had a `v-data-table` with a `v-slot="item.name"` I think. It complained about the slot name, as if it didn't recognize `item.name` and `item.age`, etc, were different slot names :man_shrugging:

Reply:
Yes, was just taking a look through the rule examples and it looks like what we have is compliant.